test_util/summary.cpp

```diff
@@ -89,11 +95,34 @@ int main(int argc, char **argv) {
 
     int argOffset = optind;
 
+    std::unique_ptr<Opm::EclIO::ESmry> emsry;
+    std::unique_ptr<Opm::EclIO::ExtESmry> ext_emsry;
```

Comment:
Tiny nit: Maybe we'd want to name these `esmry` instead of `emsry` (letters `m` and `s` swapped).

Reply:
thanks, fixed typo. This is now ready from my end.

Reply:
> fixed typo.

Is there an update missing?  As far as I can tell, the current sources still uses `eMSry` instead of `eSMry`.

Reply:
Thank you for your patience. I'm not sure what happened, but should be updated now.